var/spack/repos/builtin/packages/py-torch-geometric/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyTorchGeometric(PythonPackage):
+    """PyTorch Geometric (PyG) is a geometric deep learning extension
+    library for PyTorch.  It consists of various methods for deep
+    learning on graphs and other irregular structures, also known as
+    geometric deep learning, from a variety of published papers. In
+    addition, it consists of an easy-to-use mini-batch loader for many
+    small and single giant graphs, multi gpu-support, a large number
+    of common benchmark datasets (based on simple interfaces to create
+    your own), and helpful transforms, both for learning on arbitrary
+    graphs as well as on 3D meshes or point clouds."""
+
+    homepage = "https://github.com/rusty1s/pytorch_geometric"
+    url      = "https://github.com/rusty1s/pytorch_geometric/archive/1.6.0.tar.gz"
+
+    version('1.6.0', sha256='7d5231cdcc2ebd4444f406cbf1537eb49bf90ab6f446eaf1b7af5cdbe105f3c9')
+
```

Comment:
Missing setuptools dependency

Reply:
Also, depends on Python 3.6+

Reply:
Can you check the build log and see if it uses `pytest-runner` or automatically downloads and installs it? I'm always wary of `setup_requires` dependencies, so I usually add them as build deps.

Reply:
It wasn't pulling it down, but added it as a dep anyways.